hudi-client/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -51,6 +52,11 @@
   // TODO: On by default. Once stable, we will remove the other mode.
   public static final String BLOOM_INDEX_BUCKETIZED_CHECKING_PROP = "hoodie.bloom.index.bucketized.checking";
   public static final String DEFAULT_BLOOM_INDEX_BUCKETIZED_CHECKING = "true";
+  public static final String BLOOM_INDEX_FILTER_TYPE_PROP = "hoodie.bloom.index.filter.type.prop";
+  public static final String DEFAULT_BLOOM_INDEX_FILTER_TYPE_PROP = SimpleBloomFilter.TYPE_CODE;
```

Comment:
we have a certain convention to naming property and the default. please follow the same to keep consistent.

Reply:
just "DEFAULT" as suffix to actual prop name right? I followed the same.